Example/Workmanager/ViewController.swift

```diff
@@ -1,24 +1,138 @@
-//
-//  ViewController.swift
-//  Workmanager
-//
-//  Created by Olivier Scalais on 10/29/2019.
-//  Copyright (c) 2019 Olivier Scalais. All rights reserved.
-//
-
 import UIKit
+import Workmanager
+import BackgroundTasks
 
 class ViewController: UIViewController {
 
+    // MARK: UI
+
+    private let stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.spacing = 10
+        stackView.alignment = .center
+        return stackView
+    }()
+
+    private let explanationLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 14, weight: .light)
+        label.text = "Registering a task will randomly change the corresponding ball's backgroundcolor."
```

Comment:
Very nice 👍 Maybe add a short readme explaining how to 'use' the demo project with a link to Apple's docs on how to simulate the background tasks